app/src/main/java/com/github/se/signify/ui/Utils.kt

```diff
@@ -539,12 +539,12 @@ fun LearnedLetterList(lettersLearned: List<Char>) {
 /**
  * A reusable composable function that creates the streak counter.
  *
- * @param days An int value for the number of days (streak).
+ * @param days A long value for the number of days (streak).
  * @param daysText A boolean value => true will display the string " days" after the number.
  */
 @Composable
-fun StreakCounter(days: Int, daysText: Boolean) {
-  val text = if (daysText) " days" else ""
+fun StreakCounter(days: Long, daysText: Boolean) {
+  val text = if (!daysText) "" else if (days == 1L) " day" else " days"
```

Comment:
A change in the call of `StreakCounter()` should be address in `AccountInformation()`.
You change the logic with the negation and now it don't show correctly the days in `ProfileScreen`, `FriendsListScreen` and `MyStatsScreen`.